docs/user_guide/SIMP_Administration/Classification_and_Data/SIMP_Compliance_Engine.inc

```diff
@@ -0,0 +1,112 @@
+.. _SIMP Compliance Engine:
+
+The SIMP Compliance Engine
+--------------------------
+
+The :term:`SIMP Compliance Engine` allows users to enable both Puppet
+parameter-level enforcement and validation of compliance with regulatory and
+internal policies.
```

Comment:
'parameter-level enforcement'  a bit unclear?  Sentence may read better with 'allows user to enable enforcement and validation....' and then follow with explanatory text about enforcement and validation being tied to Puppet parameters?

Reply:
I can't make this make any more sense without making it misleading. Happy for anyone else to try after my next update.

Reply:
I agree with @lnemsick-simp, its a bit unclear. Maybe you can follow the sentence with an example (i'm all about examples ;-) )

Reply:
Tried to resolve this, a re-review would be appreciated.